From 9343a5b0afb8265e02798c48ab52758d6cdfddf9 Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio@gentoo.org>
Date: Tue, 18 Aug 2020 09:53:38 +0300
Subject: [PATCH 6/6] build: Don't build gcr gtk-doc without ui enabled

gcr gtk-doc includes both gcr and gcr ui documentation, so we can't build
it without ui enabled
---
 docs/meson.build | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/docs/meson.build b/docs/meson.build
index f353516..8984ef2 100644
--- a/docs/meson.build
+++ b/docs/meson.build
@@ -1,2 +1,4 @@
 subdir('reference/gck')
-subdir('reference/gcr')
+if get_option('gtk')
+  subdir('reference/gcr')
+endif
-- 
2.20.1

